cpp/ql/src/semmle/code/cpp/models/implementations/Allocation.qll

```diff
@@ -82,7 +82,9 @@ private class AllocaAllocationFunction extends AllocationFunction {
     hasGlobalName([
         // --- stack allocation
         "alloca", // // alloca(size)
-        "__builtin_alloca" // __builtin_alloca(size)
+        "__builtin_alloca", // __builtin_alloca(size)
+        "_alloca", // __alloca(size)
```

Comment:
Is it a mistake that the `"_alloca"` string with 1 underscore when the comment has 2?

Reply:
Good spot.  One underscore is correct (see https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/alloca?view=msvc-160), I have corrected the comment.